packages/testharness/src/utils/runAsyncInterval.js

```diff
@@ -0,0 +1,15 @@
+const globalSetTimeout = setTimeout;
```

Comment:
why do we need `globalSetTimeout`?

Reply:
`setTimeout` may be overwritten by lolex (fake timers).

So we save it before it get overwritten.

I added comments in the file for explanation.